libkbfs/tlf_journal.go

```diff
@@ -460,6 +479,10 @@ func (j *tlfJournal) flushOneBlockOp(ctx context.Context) (bool, error) {
 		return false, err
 	}
 
+	// TODO: There is a race condition if we shut down after we've
+	// flushed an op but before we remove it. Make sure we handle
+	// re-flushed ops idempotently.
```

Comment:
On 09/08/2016 09:13 PM, Fred Akalin wrote:

> In libkbfs/tlf_journal.go
> <https://github.com/keybase/kbfs/pull/295#discussion_r78126621>:
>
> > @@ -460,6 +479,10 @@ func (j *tlfJournal) flushOneBlockOp(ctx context.Context) (bool, error) {
> >  		return false, err
> >  	}
> >  
> > +	// TODO: There is a race condition if we shut down after we've
> > +	// flushed an op but before we remove it. Make sure we handle
> > +	// re-flushed ops idempotently.
>
> This TODO is basically just to verify that BServer ops are all
> idempotent. We talked about this, and they should be, right?
>


Yes, so I think you can remove the TODO.